sample-multi-module/team-props/static-analysis.gradle

```diff
@@ -34,11 +34,12 @@ staticAnalysis {
     }
 
     detekt {
-        profile('main') {
-            input = project.file("src/main/java")
-            config = rootProject.file('team-props/detekt-config.yml')
-            filters = '.*test.*,.*/resources/.*,.*/tmp/.*'
-            output = project.file("build/reports/detekt")
+        input = project.files("src/main/java")
+        config = rootProject.files('team-props/detekt-config.yml')
+        filters = '.*test.*,.*/resources/.*,.*/tmp/.*'
+        reports {
+            xml.enabled = true
```

Comment:
@tasomaniac I was wondering: shouldn't we add this in `DetektConfigurator`? We actually switch both xml and html reports on in other tools too.